tests/parm/control_ca.nml.IN

```diff
@@ -181,15 +181,24 @@ deflate_level=1
   do_ca          = @[DO_CA]
   ca_sgs         = @[CA_SGS]
   ca_global      = @[CA_GLOBAL]
-  nlives         = @[NLIVES]
-  nseed          = @[NSEED]
-  nthresh        = @[NTHRESH]
-  ca_trigger     = .true.
-  ca_smooth      = @[CA_SMOOTH]
-  nspinup        = @[NSPINUP]
-  iseed_ca       = @[ISEED_CA]
-  nsmooth        = @[NSMOOTH]
-  ca_amplitude   = @[CA_AMPLITUDE]
+  nca            = 1
```

Comment:
I want to confirm the updated setting will only be used when do_ca is .true., correct? The RT test control_lndp is also using the control_ca.nml.IN as namelist template, just want to make sure there is no impact on that test.

Reply:
That is correct, thanks for checking. Results only change if do_ca = True. New baselines are needed for the tests coupled and atmosphere only tests:

Test 004 cpld_ca 
Test 026 control_ca